core/src/org/javarosa/core/model/FormDef.java

```diff
@@ -98,7 +98,7 @@
     // This list is topologically ordered, meaning for any tA
     // and tB in the list, where tA comes before tB, evaluating tA cannot
     // depend on any result from evaluating tB
-    public Vector<Triggerable> triggerables;
+    private Vector<Triggerable> triggerables;
```

Comment:
Don't pull until I add a commit to schema-gen code that makes use of this being public.